code/__HELPERS/unsorted.dm

```diff
@@ -1538,6 +1539,16 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 
 	SpinAnimation(rotation_speed, -1, clockwise, rotation_segments)
 
+	// Only create the list if we're adding a new item, so we don't have a bunch of empty lists on /every/ atom
+	if (!A.orbiters)
```

Comment:
we don't put spaces after `if` `while`, etc.

Reply:
Wasn't aware of the lazylist defines; those certainly clean up what I was trying to do by a lot.